runtime/src/gpu/cuda/gpu-cuda.c

```diff
@@ -159,6 +175,13 @@ static void chpl_gpu_launch_kernel_help(int ln,
 
   CHPL_GPU_DEBUG("Synchronization complete %s\n", name);
 
+  // free GPU memory allocated for kernel parameters
+  for (i=0 ; i<nargs ; i++) {
+    if (gpu_alloc_map[i] == 1) {
+      chpl_gpu_mem_free(*kernel_params[i], 0, 0);
```

Comment:
Not that we'd ever expect this to fail but we have `ln` and `fn` passed into `chpl_gpu_launch_kernel_help`, might as well forward them along here.